src/Tests.IntegrationTests/ClassifyMessageTests.cs

```diff
@@ -70,6 +80,9 @@ public async Task GivenValidMessageReceivesValidResponse()
             // Determine the newest message (assume this is from Dmr)
             var dmrMessage = resultChat.Messages.OrderByDescending(c => c.CreatedAt).First();
 
+
+            _output.WriteLine($"resultChat = {resultChat}");
```

Comment:
I think this might not be outputting anything useful? It might writing out the type name or something but not the internal properties.  Please remove if this is no longer needed or replace it with the ID of the message from the DMR?

Reply:
Thanks, have changed it. It was completely the wrong variable.